src/components/organisms/FileTreePane/FileTreePane.tsx

```diff
@@ -434,7 +423,7 @@ const FileTreePane = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [highlightNode]);
 
-  useEffect(() => {
```

Comment:
This would not run properly as the tree was not yet present, the code would run while we were showing "Get started by selecting a folder containing manifests, kustomizations or Helm Charts." and as a result it did nothing.